test/postgresql/deepdive_initdb.bats

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bats
+# Tests for initdb
```

Comment:
Maybe it's more important to test the python script for generating CREATE TABLE statements?